content/concepts/sampling_mode.md

```diff
@@ -37,7 +37,7 @@ To test sampling mode, set the sampling percentage to 50 (which represents 50%),
 $ curl -v http://localhost/index.html?test=/etc/passwd
 ```
 
-* If the Core Rule Set is applied to the transaction *(and the inbound anomaly threshold is set to 10 or lower)* then a `403 Forbidden` status code will be returned, since the request causes two critical rules to match, by default.
+* If the CRS is applied to the transaction *(and the inbound anomaly threshold is set to 10 or lower)* then a `403 Forbidden` status code will be returned, since the request causes two critical rules to match, by default.
```

Comment:
```suggestion
* If CRS is applied to the transaction *(and the inbound anomaly threshold is set to 10 or lower)* then a `403 Forbidden` status code will be returned, since the request causes two critical rules to match, by default.
```